tests/acceptance/config/behat.yml

```diff
@@ -8,11 +8,7 @@ default:
         - '%paths.base%/../../../../../tests/acceptance/features'
       context: &common_ldap_suite_context
         parameters:
-          ldapAdminUser: cn=admin,dc=owncloud,dc=com
           ldapAdminPassword: admin
-          ldapBaseDN: dc=owncloud,dc=com
```

Comment:
We do not use this anywhere, do we?

Reply:
that setting is needed for setting the LDAP connection see https://github.com/owncloud/user_ldap/pull/415/files#diff-1fd6e7b9542409bf830dd808e362648aL422
before we used the setting from this yml, now from the current LDAP setting see https://github.com/owncloud/user_ldap/pull/415/files#diff-1fd6e7b9542409bf830dd808e362648aR438